video-colorization/tcvc/README.md

```diff
@@ -0,0 +1,59 @@
+# TCVC Example by NNabla
+This is a NNabla implementation of the "Automatic Temporally Coherent Video Colorization" (TCVC).
+
+This work is done by Mingcheng Yuan during his internship at Sony R&D in 2021
+
+## Requirements
+
+### python environments
+You can install all dependencies from `requirements.txt`.
+```bash
+pip install -r ./requirements.txt
+``` 
+
+Note that `nnabla-ext-cuda110`(requiring CUDA 11.0.3 and CUDNN 8.0.5) will be installed in default settings.  
+If you've installed other version of CUDA, please modify `requirements.txt` to meet your environment.  
+For more information, see [NNabla CUDA extension package installation guild](https://nnabla.readthedocs.io/en/latest/python/pip_installation_cuda.html).
+
+## Dataset
+Please prepare the dataset by extracting frames (require `ffmpeg`, currently won't work when installed with pip but works with conda) from anime episodes with 
+```
+python -m external.extract_video_frames --input-path path-to-video 
+```
+and extract the line-art with "[sketchKeras](https://github.com/lllyasviel/sketchKeras)"
+
+
+## Running Code
+### Train with the original tcvc (or reference-based version)
+```
+# by a single GPU
+./scripts/generate_tcvc.sh
+or
+./scripts/generate_tcvc_feat.sh
```

Comment:
Please describe clearly what is different when you have `_feat` in the filename.